ddtrace/contrib/asgi/middleware.py

```diff
@@ -102,7 +102,7 @@ async def __call__(self, scope, receive, send):
         )
 
         if self.span_modifier:
-            self.span_modifier(span, scope)
+            self.span_modifier(span, scope, self.integration_config)
```

Comment:
This will be a breaking change for starlette users that have a custom `span_modifier`. Can we pull this change into a separate PR as it'll need a separate release note and tests?